app/models/hxl_stats_view.rb

```diff
@@ -0,0 +1,109 @@
+class HxlStatsView < ActiveRecord::Base
+  # View aggregates data in a multi-nested-group by for the HXL proxy/endpoint
+  #
+  self.table_name = 'hxl_stats_view'
+  def readonly?
+    true
+  end
+
+  HXLSTATS_COLUMN_HEADERS =
+    ["Emotional State",
+     "Stage of journey",
+     "Country drawn in",
+     "Total children affected",
+     "Children who identify as female",
+     "Children who identify as male",
+     "Children who identify as neither female nor male",
+     "Children between the ages of 5-12",
+     "Children between the ages of 13-18",
+     "Children under 5 years old",
+     "Older than 18 years old"].freeze
+
+  HXL_STATS_TAGS =
+    ["#impact+indicator+code",
+     "#affected+children",
+     "#country+code",
+     "#affected+children+total",
+     "#affected+children+female",
+     "#affected+children+male",
+     "#affected+children+other_gender",
+     "#affected+children+age_5_12",
+     "#affected+children+age_13_18",
+     "#affected+children+age_under5",
+     "#affected+children+age_18plus"].freeze
+
+  private_class_method :get_counts_by_gender
+  private_class_method :get_counts_by_age
+
+  def self.get_counts_by_gender(hxlstatsmajorgroup)
+    # Group by gender and aggregate
+    gender_totals = Hash.new(0)
+    hxlstatsgroupsgender = hxlstatsmajorgroup.group_by(&:gender) # shorthand for { |hxlstat| hxlstat.gender }
+    hxlstatsgroupsgender.each do |k_gender, v_gender|
+      # puts "    cycle genders"
+      # puts "    %s " % k_gender
+      # puts "    %s " % v_gender
+      gender_totals[k_gender] = v_gender.count
+    end
+    gender_totals # hash of key gender to v counts
+  end
+
+  # rubocop:disable MethodLength
```

Comment:
Rubocop doesn't need this methodlength exception for this method.

Also, you can run rubocop locally without having to wait for CI to build. Just cd into the root of the project and type `rubocop`. Or you can do it for a specific file by typing the path after it.